docs/src/systems/dem.md

```diff
@@ -17,17 +17,3 @@ and moments acting upon them.
 Modules = [TrixiParticles]
 Pages = [joinpath("schemes", "solid", "discrete_element_method", "system.jl")]
 ```
-
-## References
-- N. Bićanić. "Discrete element methods". 
-  In: Encyclopedia of Computational Mechanics (2007).
-  [doi: 10.1002/0470091355.ecm006.pub2](https://doi.org/10.1002/0470091355.ecm006.pub2)
-
-- P. Cundall and O. Strack. "A discrete numerical model for granular assemblies". 
-  In:  Géotechnique 29.1 (1979), pages 47--65.
-  [doi: 10.1680/geot.1979.29.1.47](https://doi.org/10.1680/geot.1979.29.1.47)
-
-- A. Renzo and F. Maio. "Comparison of contact-force models for the simulation of collisions in DEM-based granular flow codes"
-  In: Chemical Engineering Science 59.3 (2004), pages 525--541.
-  [doi: 10.1016/j.ces.2003.09.037](https://doi.org/10.1016/j.ces.2003.09.037)
```

Comment:
These references didn't appear in the text above, so I didn't know where to put them.

Reply:
Then we will have to add them to the text.